corehq/apps/user_importer/importer.py

```diff
@@ -727,10 +733,14 @@ def process_existing_user(self, user):
                     membership, role_qualified_id, user, web_user_importer
                 )
             else:
+                profile = None
+                if self.column_values["profile_name"] in self.domain_info.profiles_by_name:
```

Comment:
Does this one also need user feedback if the profile isn't found?

Reply:
yes it does [34215de](https://github.com/dimagi/commcare-hq/pull/34466/commits/34215de14cfdea0340b31f3b2b860ee0b7491a5c)